includes/ucf-degree-utils.php

```diff
@@ -185,4 +185,40 @@ function ucf_degree_add_query_args( $args, $request ) {
 	add_filter( 'rest_degree_query', 'ucf_degree_add_query_args', 10, 2 );
 }
 
+if ( ! function_exists( 'ucf_degree_add_recurrence_intervals' ) ) {
+	function ucf_degree_add_recurrence_intervals( $schedules ) {
+		if ( ! isset( $schedules['daily'] ) ) {
+			$schedules['daily'] = array(
+				'interval' => 24 * 60 * 60, // 24 hours * 60 minutes * 60 seconds
```

Comment:
Just for future reference, WordPress provides constants specifically for expressing time in seconds, e.g. `DAY_IN_SECONDS`:  https://codex.wordpress.org/Easier_Expression_of_Time_Constants